_data/definitions.yml

```diff
@@ -0,0 +1,14 @@
+BOVS:
+  Node: A vertex in a BOVS diagram. Nodes function like nouns, representing Parties.
+  Arrow: An edge in a BOVS diagram. Arrows function like verbs, connecting Nodes to show relationships.
+  Chain: A series of Nodes continuously connected by Arrows is called a Chain. The "start" of the Chain is the first Node and the "end" is the last, based on the direction of the Arrows.
+  Party: General term for any kind of thing (e.g. Person, Company) that can be involved in a beneficial ownership situation.
+  Beneficiary: Any Party that has ownership or control of another Party. The Beneficiary at the start of a Chain is the "ultimate" Beneficiary.
+  Subject: Any type of Party that cannot be legally owned or controlled (e.g. a Person). Subjects are always the ultimate Beneficiaries, i.e always appear at the start of a Chain.
```

Comment:
Ah. We've got a conflict here with the way BODS uses 'subject'. See http://standard.openownership.org/en/0.2.0/schema/reference.html?highlight=Party#ownership-or-control-statement.json,,subject 

In BODS, the Subject is always an entity, and occupies the positions labelled 'object' in the current diagram: 

![image](https://user-images.githubusercontent.com/342624/84668731-dd4cd800-af1b-11ea-8d8e-41c65661f56e.png)

I realise this is at odds with the natural way of talking about BOVS diagrams when we think of a graph as a set of directed relationships from Subject -> Predicate -> Object. However, I think it is important we avoid an inconsistency with BODS here.

BODS also avoids the term **'object'** as in 'Subject, Predicate, Object', because in schema language, object has another use (to refer to a schema element that can contain other fields, rather than only literal values).

Even though it lacks some of the elegance of the current use of the terms, I would propose we:

* Replace the term "Beneficiaries" with "Interested Parties" (following BODS language) - as this conceptually maps to the current labelled 'Beneficiaries' bracket on the diagram above.

* Switch the term "Object" to be "Subject" and define as "Any type of Party that may be legally owned or controlled (e.g. a Company)." 

This would give the following definitions at https://deploy-preview-39--open-ownership.netlify.app/visualisation/core/definitions/:

* **Party** - General term for any kind of thing (e.g. Person, Company) that can be involved in a beneficial ownership _relationship_.

* **Interested Party** - Any Party that has an ownership or control interest in another Party. 

* **Beneficial Owner** - Any Person at the start of a Chain, by virtue of having ownership and control interests which are not, in turn, owed to, or controlled by, another Person." 

* **Subject** - Any type of Party that must be legally owned or controlled (e.g. a Company). Subjects can appear at any point in a Chain except the start.

* **Direct/Indirect Interest**
A Party has a Direct interest in another if the Distance between them is 1. Otherwise the interest is Indirect.


This has implications for language elsewhere in the pages - particularly https://deploy-preview-39--open-ownership.netlify.app/visualisation/core/focus-depth/

Reply:
@timgdavies A couple of thoughts:

* Is the term "Beneficial Owner" potentially confusing as this Party may actually only _control_, not own? – I would prefer Beneficiary (and "ultimate Beneficiary") for this reason, or similar.
* Looking at how "subject" is used in BODS, my preference would be to avoid the clash altogether. I don't think referring to Entities lower down Chains as "Subjects" adds clarity here. I'd prefer we pick a different term entirely; and possibly we don't even need one; can we actually just say "Entity"? – which presumably is the most generic term that will apply to anything lower down a Chain?

Reply:
@Convincible - just replying in Tim's absence so you're not blocked :-)

> Is the term "Beneficial Owner" potentially confusing as this Party may actually only control, not own?
The term has been adopted in this domain - not just by us - to explicitly encompass control as well as legal ownership. So we should use it and be explicit about its full scope.

On the second point - if you can avoid the clash by picking another term ('entity' might work) that's good. If we can get away with not using any specific term or classification for 'the-thing-down-chain'  then even better.

Reply:
Thanks Kadie. Can you confirm that once we're past Beneficial Owners, everything further along the Chain will always be a kind of Entity? (That is, Entity is already the most generic term for a thing you can have an interest in.)

Reply:
@Convincible - yes, that's right.

Reply:
This all looks good to me now, so resolving this